content/applications/inventory_and_mrp/inventory/inventory_valuation/landed_costs.rst

```diff
@@ -88,15 +88,16 @@ Once the vendor fulfills the |PO| and sends a bill, a vendor bill can be created
 Odoo.
 
 Navigate to the :menuselection:`Purchase app`, and click into the |PO| for which a vendor bill
-should be created. Then, click :guilabel:`Create Bill`. This opens a new :guilabel:`Vendor Bill` in
-the :guilabel:`Draft` stage.
+should be created. Then, upload the bill or click the :guilabel:`Bill Matching` smart button to
+match with an existing bill. This opens a new :guilabel:`Vendor Bill` in the :guilabel:`Draft`
+stage.
 
 In the :guilabel:`Bill Date` field, click the line to open a calendar popover menu, and select the
 date on which this draft bill should be billed.
 
-Then, under the :guilabel:`Invoice Lines` tab, click :guilabel:`Add a line`, and click the drop-down
-menu in the :guilabel:`Product` column to select the previously-created landed cost product. Click
-the :icon:`fa-cloud-upload` :guilabel:`(cloud with arrow)` icon to manually save and update the
+Click :guilabel:`Add a line`. Add the landed cost product to the vendor bill.
+
+Click the :icon:`fa-cloud-upload` :guilabel:`(Save manually)` icon to manually save and update the
```

Comment:
optional suggestion, per the [UI appendix](https://docs.google.com/document/d/17UxnITmNyV017LbvqEGmhy6vxICDwkiDuzaE_b40BbE/edit?tab=t.0#heading=h.tc1oju2fzow0)

```suggestion
Click the :icon:`fa-cloud-upload` :guilabel:`(Save)` icon to manually save and update the
```